test/dialects/dialects_test.py

```diff
@@ -22,30 +25,94 @@
 )
 
 
-@pytest.mark.parametrize("dialect,file", parse_success_examples)
-def test__dialect__base_file_parse(dialect, file):
-    """For given test examples, check successful parsing."""
-    raw = load_file(dialect, file)
+def _lex_and_parse(config_overrides: str, raw: str) -> Optional[BaseSegment]:
+    """Performs a Lex and Parse, best cached within fixture."""
     # Load the right dialect
-    config = FluffConfig(overrides=dict(dialect=dialect))
+    config = FluffConfig(overrides=json.loads(config_overrides))
     tokens, lex_vs = Lexer(config=config).lex(raw)
     # From just the initial parse, check we're all there
     assert "".join(token.raw for token in tokens) == raw
     # Check we don't have lexing issues
     assert not lex_vs
-
     # Do the parse WITHOUT lots of logging
     # The logs get too long here to be useful. We should use
     # specific segment tests if we want to debug logs.
```

Comment:
`pytest test/dialects/dialects_test.py | grep -v "INFO\|DEBUG\|\[L\|#\|Initial\|^$"`
Really helps with the overly verbose logging that occurs during errors here. Should we put it somewhere for people connivence? eg in a comment here.
Excluding lines that:
- have INFO
- have DEBUG
- output the parse tree ([L ....
- mention Initial Structure (Initial)
- describe parse depth -  # Beginning Parse Depth 4: SelectStatementSegment
- Are blank `^$`

Reply:
One potential alternative: Could we use `caplog` to temporarily reduce the logging verbosity?

https://docs.pytest.org/en/6.2.x/logging.html#caplog-fixture